_includes/javascripts.html

```diff
@@ -1,9 +1,13 @@
 {% unless site.local_fonts %}
-  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/webfont/1.6.27/webfontloader.js"></script>
```

Comment:
We don't store the google analytics one as well. Any reason why we should do that? The CDN of cloudflare is probably the fastest way :D Any other reason why we don't use the googleapis one?